.github/workflows/publish-patch-manual.yml

```diff
@@ -0,0 +1,154 @@
+name: Manually Triggered Patch Release
```

Comment:
Note: A patch on stable VSCode extension has to be triggered manually. There can be two cases

1. A patch branch already exists (.x) because of Prisma CLI and we just need to manually trigger this workflow
2. We are not patching the CLI and we need to create the respective `.x` branch, push to that to branch to release insider extension to manually test and when satisfied, manually trigger this workflow